azure-monitor-opentelemetry-distro/samples/logging/correlated_logs.py

```diff
@@ -0,0 +1,26 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from logging import WARN, getLogger
```

Comment:
I think `WARN` is actually deprecated so we should be using `logging.WARNING`.

Reply:
Done.